faq/running.inc

```diff
@@ -1634,6 +1634,32 @@ $a[] = "See these two FAQ categories:
 
 /////////////////////////////////////////////////////////////////////////
 
+$q[] = "What are the libfabric (OFI) components in Open MPI?";
+
+$anchor[] = "mpi-ofi-support";
+
+$a[] = "
+Open MPI has two main components for Libfabric (a.k.a OFI) communications:
+
+*OFI MTL:* For two sided tag matching MPI communication. Included since v1.10.
+Works with providers that support:  EP type: FI_EP_RDM; Capabilities: FI_TAGGED ; Ordering: FI_ORDER_SAS
+
+*OFI BTL:* For one sided communications. Included since v4.0.
+Works with providers that support:  EP type: FI_EP_RDM; Capabilities: FI_RMA and FI_ATOMIC; Op flags: FI_DELIVERY_COMPLETE;
+This BTL is expected to be used with OSC RDMA.
```

Comment:
I suggest changing these two to:

```
<ol>
<li> `ofi` MTL: Available since Open MPI v1.10, this component is used with the `cm` PML and is likely used by default for two-sided MPI communication (e.g., MPI_SEND and MPI_RECV).  The `ofi` MTL requires that the Libfabric provider support reliable datagrams with ordered tagged messaging (specifically: `FI_EP_RDM` endpoints, `FI_TAGGED` capabilities, and `FI_ORDER_SAS` ordering).</li>
<li> `ofi` BTL: Available since Open MPI v4.0.0, this component is used for one-sided MPI communications (e.g., MPI_PUT).  The `ofi` BTL requires that the Libfabric provider support reliable datagrams, RMA and atomic operations, and remote atomic completion notifications (specifically: `FI_EP_RDM` endpoints, `FI_RMA` and `FI_ATOMIC` capabilities, and `FI_DELIVERY_COMPLETE` op flags).</li>
</ol>
```